include/freerdp/channels/rdpdr.h

```diff
@@ -298,6 +298,7 @@ enum FILE_FS_DEVICE_FLAG
 	FILE_DEVICE_SECURE_OPEN = 0x00000100
 };
 
+#ifndef __MINGW32__
```

Comment:
Might be a good idea to rename the struct as it might not be equal to what we expect in freerdp

Reply:
`FILE_FS_INFORMATION_CLASS` is defined in mingw-w64 but not in Microsoft Windows SDK. So, this may be the right choice.